read-fonts/src/tables/postscript/charstring.rs

```diff
@@ -78,6 +78,179 @@ where
     }
 }
 
+/// Command sink adapter that applies a scaling factor.
+///
+/// This assumes a 26.6 scaling factor packed into a Fixed and thus,
+/// this is not public and exists only to match FreeType's exact
+/// scaling process.
+pub(crate) struct ScalingSink26Dot6<'a, S> {
+    inner: &'a mut S,
+    scale: Fixed,
+}
+
+impl<'a, S> ScalingSink26Dot6<'a, S> {
+    pub fn new(sink: &'a mut S, scale: Fixed) -> Self {
+        Self { scale, inner: sink }
+    }
+
+    fn scale(&self, coord: Fixed) -> Fixed {
+        if self.scale != Fixed::ONE {
+            // The following dance is necessary to exactly match FreeType's
+            // application of scaling factors:
```

Comment:
if it's possible to tell, perhaps note _why_ we think FT does it the way it does it. Or if not perhaps note that :)

Reply:
My best guess is that this is a combination of FT being a TrueType first implementation and the fact that the Adobe code was added later… so it was all plugged in to fit the existing model. I’ll note that this was likely done for historical and compatibility reasons 🤷🏻‍♂️

Reply:
Done